.github/workflows/ci-format-private.yml

```diff
@@ -0,0 +1,30 @@
+# This is a format job. Pre-commit has a first-party GitHub action, so we use
```

Comment:
This is a public, not private repository... Please adjust to public repositories template